### PR TITLE
Aduminuc bfla

### PR DESCRIPTION
**Pull request by aduminuc**
Fix BFLA testing
Fix bFLA linting

### CODE REVIEW

File: backend/pkg/modules/internal/bfla/bfladetector/opeanapi_provider.go [hunk 1]
```diff
@@ -17,6 +17,7 @@ package bfladetector
 
 import (
 	"fmt"
+
```
[the remark this answers is not in the document]
Comment (aduminuc) [reply backend/pkg/modules/internal/bfla/bfladetector/opeanapi_provider.go:20]: it's the linter

[the remark this answers is not in the document]
Comment (aduminuc) [reply backend/pkg/modules/internal/bfla/bfladetector/opeanapi_provider.go:20]: I think

File: backend/pkg/modules/modules.go [hunk 2]
```diff
@@ -27,15 +27,15 @@ import (
 	"github.com/openclarity/apiclarity/backend/pkg/modules/internal/core"
 
 	// Enables the demo module.
-	// _ "github.com/openclarity/apiclarity/backend/pkg/modules/internal/demo"
+	// _ "github.com/openclarity/apiclarity/backend/pkg/modules/internal/demo" .
```
[the remark this answers is not in the document]
Comment (aduminuc) [reply backend/pkg/modules/modules.go:30]: Linter

File: backend/pkg/modules/internal/bfla/bfladetector/opeanapi_provider.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (danielvladco) [reply backend/pkg/modules/internal/bfla/bfladetector/opeanapi_provider.go:20]: I thought it's 2 lines... so it's good, just regroup the imports so they are grouped in 3 main groups

File: backend/pkg/modules/internal/fuzzer/model/model.go [hunk 3]
```diff
@@ -146,7 +146,7 @@ func (m *Model) StartAPIFuzzing(ctx context.Context, apiID uint, params *Fuzzing
 	if err != nil {
 		return ZeroTime, fmt.Errorf("can't start fuzzing (%v)", apiID)
 	}
-	//dumpSlice(m.db)
+	// dumpSlice(m.db)
```
[the remark this answers is not in the document]
Comment (aduminuc) [reply backend/pkg/modules/internal/fuzzer/model/model.go:149]: I will let Axel do it as it is in the fuzzer

**Review by danielvladco — APPROVED** [review]

**Pull request merged**